### PR TITLE
sync with english version for void element

### LINKED ISSUE #14424: zh-CN: void element has different definitions with English version
### MDN URL

https://developer.mozilla.org/zh-CN/docs/Glossary/Void_element

### 与此议题相关的小节或标题？

Void element

### 哪些信息是错误、无用或不完善的？

Translations about Void element.

### 你期望看到什么？

`Void element` is called `empty element` in the Chinese version. And it points out that `In rare cases, empty elements are mistakenly referred to as "void elements".` But there is no term "empty element" in the English version. Seems that the Chinese version is outdated. Additionally, the Chinese version is missing content on `Self-closing tags`.

Void element在中文里被翻译作empty elements，并且备注说`在极少数情况下，空元素被错误地称为“无效元素”(void elements)。`但是我看英文版里没有empty element这个说法。这里的翻译似乎过时了？另外中文翻译里缺少`Self-closing tags`这块的内容。

### 你有什么支持的链接、参考或引用？

_No response_

### 你还想补充什么内容？

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `zh-cn/glossary/void_element`
* MDN URL: https://developer.mozilla.org/zh-CN/docs/Glossary/Void_element
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/zh-cn/glossary/void_element/index.md
* Last commit: https://github.com/mdn/translated-content/commit/24f265c3f3bb208770dc304853c4afe9d96901e3
* Document last modified: 2023-06-14T14:20:21.000Z

</details>

### PR DESCRIPTION
**Pull request by yin1999**
### Description

sync with english version for void element

### Related issues and pull requests

Fixes: #14424

### CODE REVIEW

File: files/zh-cn/glossary/void_element/index.md [hunk 1]
```diff
@@ -5,13 +5,15 @@ slug: Glossary/Void_element
...
 {{GlossarySidebar}}
 
-一个**空元素**（empty element）可能是 HTML、SVG，或者 MathML 里的一个不能存在子节点（例如内嵌的元素或者元素内的文本）的{{Glossary("element", "元素")}}。
+**空元素**（empty element）是 HTML 中不能存在子节点（例如内嵌的元素或者文本节点）的{{Glossary("element", "元素")}}。空元素只有开始标签且不能指定结束标签。
 
-[HTML](https://www.w3.org/html/wg/drafts/html/CR/)，[SVG](https://www.w3.org/TR/SVG2/) 和 [MathML](https://www.w3.org/Math/draft-spec/) 的规范都详细定义了每个元素能包含的具体内容（define very precisely what each element can contain）。许多组合是没有任何语义含义的，比如一个 {{HTMLElement("audio")}} 元素嵌套在一个 {{HTMLElement("hr")}} 元素里。
+在 HTML 中，空元素不能有结束标签。例如，`<input type="text"></input>` 是无效的 HTML。相反，不能有任何子节点的 SVG 或 MathML 元素可以使用结束标签来代替 XML 的在开始标签中的自闭合标签语法。
 
-在 HTML 中，通常在一个空元素上使用一个闭标签是无效的。例如， `<input type="text"></input>` 的闭标签是无效的 HTML。
+[HTML](https://html.spec.whatwg.org/multipage/)、[SVG](https://www.w3.org/TR/SVG2/) 和 [MathML](https://www.w3.org/TR/MathML3/) 的规范都详细定义了每个元素能包含的具体内容。所以一些元素的组合是没有任何语义的。
```
Comment (yin1999) [inline files/zh-cn/glossary/void_element/index.md:12]:
```suggestion
[HTML](https://html.spec.whatwg.org/multipage/)、[SVG](https://www.w3.org/TR/SVG2/) 和 [MathML](https://www.w3.org/TR/MathML3/) 的规范都详细定义了每个元素能包含的具体内容。所以一些标签的组合是没有任何语义的。
```

File: files/zh-cn/glossary/void_element/index.md [hunk 2]
```diff
@@ -23,9 +25,25 @@ slug: Glossary/Void_element
...
 - {{HTMLElement("source")}}
 - {{HTMLElement("track")}}
 - {{HTMLElement("wbr")}}
 
-> **备注：** 在极少数情况下，空元素被错误地称为“无效元素”(void elements)。
+## 自闭合标签
+
+_HTML 中不存在自闭合标签（`<tag />`）。_
+
+如果一个 HTML 元素的开始标签中存在尾随的 `/`（斜杠）字符，HTML 解析器会忽略该斜杠字符。请记住一些元素（例如 {{HTMLElement('script')}} 或 {{HTMLElement('ul')}}）的确需要结束标签。在这种情况下，在开始标签中添加尾随斜杠不会关闭元素。
```
Comment (yin1999) [inline files/zh-cn/glossary/void_element/index.md:37]:
```suggestion
如果一个 HTML 元素的开始标签中存在尾随的 `/`（斜杠）字符，HTML 解析器会忽略该斜杠字符。请记住一些元素（例如 {{HTMLElement('script')}} 或 {{HTMLElement('ul')}}）的确需要结束标签。在这种情况下，在开始标签中添加尾随斜杠不会闭合元素。
```

File: files/zh-cn/glossary/void_element/index.md [hunk 3]
```diff
@@ -23,9 +25,25 @@ slug: Glossary/Void_element
...
+
+但是，某些代码格式化工具会在空元素的开始标签中添加尾随斜杠字符，以使其兼容 XHTML 并更易读。例如，某些代码格式化工具会将 `<input type="text">` 转换为 `<input type="text" />`。
+
+[XML](/zh-CN/docs/Glossary/XML)、[XHTML](/zh-CN/docs/Glossary/XHTML) 和 [SVG](/zh-CN/docs/Glossary/SVG) 的空元素则必须使用自闭合标签（例如 `<circle cx="50" cy="50" r="50" />`）。
+
+在 SVG 和 MathML 中，不允许有任何子节点的元素可以使用子闭合标签。在这种情况下，如果元素的开始标签被标记为自闭合，则该元素不能有结束标签。
```
Comment (yin1999) [inline files/zh-cn/glossary/void_element/index.md:43]:
```suggestion
在 SVG 和 MathML 中，有任何子节点的元素不允许使用子闭合标签。在这种情况下，如果元素的开始标签被标记为自闭合，则该元素不能有结束标签。
```

File: files/zh-cn/glossary/void_element/index.md [hunk 4]
```diff
@@ -5,13 +5,15 @@ slug: Glossary/Void_element
 
 {{GlossarySidebar}}
 
-一个**空元素**（empty element）可能是 HTML、SVG，或者 MathML 里的一个不能存在子节点（例如内嵌的元素或者元素内的文本）的{{Glossary("element", "元素")}}。
+**空元素**（empty element）是 HTML 中不能存在子节点（例如内嵌的元素或者文本节点）的{{Glossary("element", "元素")}}。空元素只有开始标签且不能指定结束标签。
```
Comment (jasonren0403) [inline files/zh-cn/glossary/void_element/index.md:8]:
```suggestion
**空元素**（empty element）是 HTML 中**不能**存在子节点（例如内嵌的元素或者文本节点）的{{Glossary("element", "元素")}}。空元素只有开始标签且不能指定结束标签。
```

**Review by jasonren0403 — APPROVED** [review]
LGTM 👍

**Pull request merged**